util/files/util.go

```diff
@@ -0,0 +1,53 @@
+package files
+
+import (
+	"os"
+	"path"
+	"path/filepath"
+	"strings"
+
+	"github.com/google/uuid"
+)
+
+// RelativePath will remove the basePath string from the fullPath
+// including the path separator. Will return fullPath if basePath
+// does not match (example 2).
+//
+// Example 1:
+//   fullPath: /home/test/app/readme.md
+//   basePath: /home/test
+//   return:   app/readme.md
+//
+// Example 2:
+//   fullPath: /home/test/app/readme.md
+//   basePath: /somewhere/else
+//   return:   /home/test/app/readme.md
+//
+// Example 3:
+//   fullPath: /home/test/app/readme.md
+//   basePath: /home/test/app/readme.md
+//   return:   .
+func RelativePath(fullPath, basePath string) string {
```

Comment:
This will return an absolute path for `fullPath == "/home/test//app/`. If this is going to be a security-critical function, I think we'll want to unit test the heck out of it.

Reply:
I already implemented unit tests for the basic scenarios. I would be glad to add more scenarios if that is checking for a valid case. 

What the test case you are suggesting look like??
Example:
```
	name:     "will return full path if basepath does not match",
	fullpath: "/home/test/app/readme.md",
	basepath: "/somewhere/else",
	expected: "/home/test/app/readme.md",
```

Reply:
```go
		{
			name:     "will return relative path from dir path",
			fullpath: "/home/test//app/",
			basepath: "/home/test",
			expected: "app",
		},
```

Reply:
added